services/movie.py

```diff
@@ -0,0 +1,52 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+from typing import Optional
+
+
+def get_movies(
+        genres_ids: Optional[list[int]] = None,
+        actors_ids: Optional[list[int]] = None,
+) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+        return queryset
+
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+        return queryset
+
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+        return queryset
+
+    return queryset
```

Comment:
instead of making return everywhere you can do filtering and return it at the end.
in this case you'll not need the first if statement